discovery-provider/src/tasks/playlists.py

```diff
@@ -170,6 +170,8 @@ def parse_playlist_event(
             )
             playlist_record.playlist_contents = {"track_ids": new_playlist_content_array}
             playlist_record.timestamp = block_datetime
+            # what's the above line for?
```

Comment:
i think it should be deleted! good callout. once upon a time before created_at and updated_at, this may have been used? @hareeshnagaraj  or @dmanjunath to confirm?